### ISSUE TITLE
Make Header of the Site Shorter

### ISSUE THREAD

**Issue by Brahmanathaswami** [Himalayan-Academy/hamsa#51]
![2020-02-06_09-06-17](https://user-images.githubusercontent.com/3675722/73999403-bbf23480-4908-11ea-911d-4f5497ab02d6.png)

**Comment by soapdog** [comment]
I don't understand what you mean by:

> Move Title to the right and up 
> Make the blue space not so high

That blue region is actually an image: http://dev.himalayanacademy.com/hamsa/images/hero1.jpg

**Comment by soapdog** [comment]
I have moved the image around by setting a negative `top` in the CSS, this allows us to make the hero shorter. I've also styled the placeholder for the search box to be the same as the other buttons but be aware that placeholders have transparency applied to them by the system so it is a bit faint, but they are set to the same color. Tried applying an outline to the search box but it looked really ugly and almost invisible. Users are familiar with search boxes, the way it is it is behaving like most search boxes out there. It has a placeholder, a magnifying glass icon, and the cursor changes to text input when hovered.
